packages/synthetics-sdk-broken-links/src/navigation_func.ts

```diff
@@ -436,3 +364,20 @@ export async function closeBrowser(browser: Browser) {
     if (err instanceof Error) process.stderr.write(err.message);
   }
 }
+
+/**
+ * Navigates the provided Puppeteer page to about:blank.
+ *
+ * @param {Page} page - The Puppeteer page to navigate.
+ * @returns {Promise<void>} - A Promise that resolves when the navigation is complete.
+ */
+async function gotoBlankPage(page: Page) {
+  try {
+    await page.goto('about:blank', {
+      waitUntil: 'load',
+      timeout: 3000,
```

Comment:
where does 3000 come from? seems like about:blank should be more or less instantaneous?